### ISSUE TITLE
Convert `builder.yml` into a role

### ISSUE THREAD

**Issue by leifmadsen** [redhat-nfvpe/kube-ansible#138]
There are enough task files and other variables required to be defined now that converting the contents of `builder.yml` into a role makes a lot of sense now.

Blocked by #119

**Comment by leifmadsen** [comment]
After reviewing and discussing with @dougbtv, we both agree that this is more of a nice to have for now, and that the current build out of the `builder.yml` isn't super obvious on how to break it into a role directly because of the multi-host usage.
